### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by RuibinMa** [DennisHanyuanXu/Tiny-ImageNet#2]
Hi, I'm also training an alexnet on tiny imagenet for a course project. Using both your model and the original alexnet, I got severe overfitting (validation loss increases at the 2nd epoch). I'm wondering if you found overfitting when you were training you network and what's your solution for that. Thanks!

**Comment by RuibinMa** [comment]
NVM, it turns out that I didn't realize the different structure between train and val.
